t/OptionFields.t

```diff
@@ -0,0 +1,133 @@
+# This file is part of Business:PayPal:API Module.   License: Same as Perl.  See its README for details.
+# -*- mode: cperl -*-
+use Test::More;
+use strict;
+use autodie qw(:all);
+use Cwd;
+
+if ( !$ENV{WPP_TEST} || !-f $ENV{WPP_TEST} ) {
+    plan skip_all =>
+        'No WPP_TEST env var set. Please see README to run tests';
+}
+else {
+    plan tests => 14;
+}
+
+use_ok( 'Business::PayPal::API::TransactionSearch' );
```

Comment:
I think `use_ok` is probably not a useful test here.  See http://modernperlbooks.com/mt/2012/04/use-ok-is-broken-because-require-is-broken.html